src/lib.rs

```diff
@@ -161,21 +169,21 @@ impl Element {
             namespaces: None,
             attributes: HashMap::new(),
             children: Vec::new(),
-            text: None,
         }
     }
 
     /// Parses some data into an Element
-    pub fn parse<R: Read>(r: R) -> Result<Element, ParseError> {
+    pub fn parse<R: Read>(r: R) -> Result<Vec<XMLNode>, ParseError> {
         let mut reader = EventReader::new(r);
```

Comment:
Instead of `EventReader::new`, we'll need to use `EventReader::new_with_config`, since the default configuration will ignore comments)